src/SprykerEco/Shared/Payone/Transfer/payone.transfer.xml

```diff
@@ -518,5 +532,11 @@
         <property name="customerMessage" type="string"/>
         <property name="gender" type="string"/>
     </transfer>
+
+    <transfer name="PayoneKlarnaSessionResponse">
```

Comment:
* Rename to be consistent with a request transfer object name `PayoneKlarnaStartSessionResponse`.

Reply:
++